### PR TITLE
updated

### PR DESCRIPTION
**Pull request by Fedirrr**

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -39,15 +34,24 @@ const Header = () => {
                     <span>WATCHLIST</span>
                 </Link>
                 <Link to="/movie">
-                    <img src="/images/movie-icon.svg" alt="movie"/>
+                    <img src="/images/movie-icon.svg" alt="movies"/>
                     <span>MOVIE</span>
                 </Link>
             </header.NavMenu>
+            <form style={{width: "100%",
```
Comment (VladDrawer) [inline src/components/Header.tsx:41]: please do not use inline styles

File: src/components/Header.tsx [hunk 2]
```diff
@@ -39,15 +34,24 @@ const Header = () => {
...
+                display: "contents"}}>
+
+                <header.Search type="search" placeholder="Search"
+                               value={searchTerm}
+                               autoComplete="off"
+                               onChange={handleInputChange}
+                />
+                <button type="button" onClick={handleSearch}>
+                    find
```
Comment (VladDrawer) [inline src/components/Header.tsx:50]: Please start all words with the capitalized letter

File: src/components/Header.tsx [hunk 3]
```diff
@@ -39,15 +34,24 @@ const Header = () => {
...
             </header.NavMenu>
+            <form style={{width: "100%",
+                display: "contents"}}>
+
+                <header.Search type="search" placeholder="Search"
+                               value={searchTerm}
+                               autoComplete="off"
+                               onChange={handleInputChange}
+                />
+                <button type="button" onClick={handleSearch}>
```
Comment (VladDrawer) [inline src/components/Header.tsx:49]: button type button?

File: src/components/Header.tsx (hunk 1 above)
Comment (VladDrawer) [reply src/components/Header.tsx:41]: you can remove the form tag or replace it with div

File: src/components/Movie.tsx [hunk 4]
```diff
@@ -1,17 +1,71 @@
...
+import {selectMovie, setMovies} from '../store/slices/slice';
+import {useDispatch, useSelector} from 'react-redux';
+import cssStyles from '../styles/styles';
+import {IMovie, moviesArray} from "./mock";
 
-const {movie} = cssStyles;
+const { movie } = cssStyles;
 
 function Movie() {
+    // const allMovies = useSelector(selectMovie);
```
Comment (VladDrawer) [inline src/components/Movie.tsx:11]: remove commented code

File: src/components/Header.tsx [hunk 5]
```diff
@@ -1,25 +1,20 @@
-import React, {useEffect, useState} from "react";
+import React, {useState, ChangeEvent} from "react";
 import cssStyles from "../styles/styles";
 import {Link} from "react-router-dom";
-import {selectListItem} from "../store/slices/watchList";
-import {useSelector} from "react-redux";
+import {useNavigate} from 'react-router-dom';
```
Comment (VladDrawer) [inline src/components/Header.tsx:4]: please use one single code style and put the spaces in all places like this { useNavigate }

File: src/components/Movie.tsx [hunk 6]
```diff
@@ -1,17 +1,71 @@
...
+
+    const dispatch = useDispatch();
+    useEffect(():void => {
+        dispatch(setMovies(moviesArray))
+    }, []);
+
+    useEffect(() => {
+        const searchParams = new URLSearchParams(location.search);
+        const searchTerm = searchParams.get('search');
+        console.log(searchTerm);
```
Comment (VladDrawer) [inline src/components/Movie.tsx:26]: remove all comsole.log's

File: src/components/Header.tsx (hunk 1 above)
Comment (Fedirrr) [reply src/components/Header.tsx]: did

File: src/components/Header.tsx (hunk 2 above)
Comment (Fedirrr) [reply src/components/Header.tsx]: did

File: src/components/Header.tsx (hunk 3 above)
Comment (Fedirrr) [reply src/components/Header.tsx]: did

File: src/components/Movie.tsx (hunk 4 above)
Comment (Fedirrr) [reply src/components/Movie.tsx]: did

File: src/components/Movie.tsx [hunk 7]
```diff
@@ -1,17 +1,71 @@
...
 
-const {movie} = cssStyles;
+const { movie } = cssStyles;
 
 function Movie() {
+    // const allMovies = useSelector(selectMovie);
+    const location = useLocation();
+    const [searchResults, setSearchResults] = useState<IMovie[]>([]);
+    const [noResults, setNoResults] = useState(false);
+
```
[the remark this answers is not in the document]
Comment (Fedirrr) [reply src/components/Movie.tsx]: did

File: src/components/Header.tsx (hunk 5 above)
Comment (Fedirrr) [reply src/components/Header.tsx]: did

File: src/components/Movie.tsx [hunk 8]
```diff
@@ -1,17 +1,71 @@
...
+
+
+    const dispatch = useDispatch();
+    useEffect(():void => {
```
[the remark this answers is not in the document]
Comment (Fedirrr) [reply src/components/Movie.tsx]: did

File: src/components/Movie.tsx (hunk 6 above)
Comment (Fedirrr) [reply src/components/Movie.tsx]: did

**Pull request merged**